plugins/GSdx/GSRenderer.cpp

```diff
@@ -83,6 +83,27 @@ void GSRenderer::ResetDevice()
     if(m_dev) m_dev->Reset(1, 1);
 }
 
+// Mizuiro constantly alters it's DH value on the DISP register from 464 to 480
+// This causes the screen to shake vertically due to the frequent change in DISP register value
+// This doesn't happen in the PS2, the assumption is that the GS has a delay before applying
+// a new DH value to the output circuit. The following function handles such delay.
```

Comment:
> You mean that DH value is overwritten before it is use, isn't it ?

Yes.

> What is "frequent" change for you ? Before Vsync ?

It's like this,

```
Vsync#1: 480
Vsync#2: 464
Vsync#3: 480
Vsync#4: 464
Vsync#5: 480
Vsync#6: 464
```
In my code, the issue is resolved by delaying the usage of new DH value usage by 1 Vsync. I think something like the following happens with the original hardware,

```
Vsync#1: 480
* First DH value used on the PCRTC output circuits initialization.

Vsync#2: 464
* New DH value detected, wait till next Vsync before applying it

Vsync#3: 480
* DH was updated once again in this Vsync back to the current value used by the PCRTC Circuit.
* PCRTC ignores the previous DH value (from Vsync#2) as it would eventually need to switch to the current DH value anyway.
```